README.developer.md

```diff
@@ -129,39 +146,64 @@ integration tests for hirte is described in [tests/README](./tests/README.md).
 
 ### Running
 
-At the moment the `hirtectl` binary does not implement any logic. It only prints a simple greeting and exit.
+The following sections describe how to run the built application(s) locally on one machine. For this, the assumed setup
+used is described in the first section.
 
-#### hirte
+#### Assumed setup
+
+The project has been build with the following command sequence:
+
+```bash
+meson setup builddir
+meson compile -C builddir
+meson install -C builddir --destdir bin
+```
+
+Meson will output the artifacts to `./builddir/bin/usr/local/`. This directory is referred to in the following sections
+simply as `<builddir>`.
 
-The controller can be run via:
+To allow `hirte` and `hirte-agent` to own a name on the local system D-Bus, the provided configuration
+files need to be copied (if not already existing):
 
 ```bash
-hirte
+cp <builddir>/share/dbus-1/system.d/org.containers.hirte.Agent.conf /etc/dbus-1/system.d/
+cp <builddir>/share/dbus-1/system.d/org.containers.hirte.conf /etc/dbus-1/system.d/
 ```
 
-It starts a tcp socket on all network interfaces on port `842` and accepts connections, but does not do much more at
-this point. This can be tested manually via:
+**Note:** Make sure to reload the dbus service so these changes take effect: `systemctl reload dbus-broker.service` (or
+`systemctl reload dbus.service`)
+
+#### hirte
+
+The newly built controller can simply be run via `./<builddir>/bin/hirte`, but it is recommended to use a specific
+configuration for development. This file can be passed in with the `-c` CLI option:
 
 ```bash
-nc <host> <port>
+./<builddir>/bin/hirte -c <path-to-cfg-file>
 ```
 
 #### hirte-agent
 
-A node can be run via:
+Before starting the agent, it is best to have the hirte controller already running. However, `hirte-agent` will
+try to reconnect in the configured heartbeat interval.
+
+Similar to `hirte`, it is recommended to use a dedicated configuration file for development:
 
 ```bash
-./bin/hirte-agent -n <agent name> -H <host>
+./<builddir>/bin/hirte -c <path-to-cfg-file>
 ```
 
-It creates a new D-Bus which tries to connect to the specified host. The host will print out a message if the request
-was accepted. It does not do much more at this point.
+#### hirtectl
 
-## Documentation
+The newly built `hirtectl` can be used via:
 
-For further documentation please refer to the [doc](./doc/) directory.
+```bash
+./<builddir>/bin/hirtectl COMMANDS
+```
+
+## Documentation
 
-The target architecture for this project is described in [doc/arch](./doc/arch/).
+For further documentation, including the readthedocs website, please refer to the [doc](./doc/) directory.
```

Comment:
Wouldn't it be better to use the real URL pointing to hirte documentation instead of *readthedocs website*?

Reply:
Good point. I re-purposed this section a bit to describe what the doc directory contains (since its a README for devs). Could you have another look? @mwperina